### PR TITLE
[DEV-1964] Adding flag to disable ssl cert verification

### PR DESCRIPTION
**Pull request by bushwhackr**
## Description

Added a new flag to disable ssl certificate verification.

```yaml
profile: 
  - name: name1
    ssl_verify: false  # This is defaulted to true
    api_url: https://something.com
    api_token: "token_value"
```

## Type of Change

Label this pull request to place it under the correct category in Release Notes:

|        **Pull Request Label**         | **Category in Release Notes** |
|:-------------------------------------:|:-----------------------------:|
|       `enhancement`, `feature`        |          🚀 Features          |
| `bug`, `refactoring`, `bugfix`, `fix` |    🔧 Fixes & Refactoring     |
|       `build`, `ci`, `testing`        |    📦 Build System & CI/CD    |
|              `breaking`               |      💥 Breaking Changes      |
|            `documentation`            |       📝 Documentation        |
|            `dependencies`             |    ⬆️ Dependencies updates    |

## Checklist

- [ ] I have read the [`CODE_OF_CONDUCT.md`](https://github.com/featurebyte/featurebyte/blob/main/CODE_OF_CONDUCT.md) and [`CONTRIBUTING.md`](https://github.com/featurebyte/featurebyte/blob/main/CONTRIBUTING.md) guides.
- [ ] I have written tests for the changes made.
- [ ] I have written docstrings in [NumpyDoc format](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)
- [ ] I have labeled my Pull Request correctly

### CODE REVIEW

File: featurebyte/config.py [hunk 1]
```diff
@@ -96,6 +97,7 @@ class Profile(BaseModel):
     name: str
     api_url: AnyHttpUrl
     api_token: Optional[str]
+    ssl_verify: bool = False
```
Comment (kchua78) [inline featurebyte/config.py:100]: Should it default to False?

File: tests/fixtures/config/config.yaml [hunk 2]
```diff
@@ -10,7 +10,6 @@ profile:
 
 logging:
   level: INFO
-  serialize: True
```
Comment (bushwhackr) [inline tests/fixtures/config/config.yaml:13]: I don't think we do serialization logging here.

File: featurebyte/config.py (hunk 1 above)
Comment (bushwhackr) [reply featurebyte/config.py]: was testing previously, now defaulted to True

**Review by kchua78 — APPROVED** [review]
LGTM

**Pull request merged**